src/tests/GC/Stress/Framework/RFLogging.cs

```diff
@@ -79,8 +79,19 @@ private void LogWorker()
                 }
                 catch (IOException e)
                 {
-                    ReliabilityFramework.MyDebugBreak(String.Format("LogWorker IOException:{0}", e.ToString()));
                     //Disk may be full so simply stop logging
+                    if (ReliabilityFramework._debugBreakOnTestHang)
```

Comment:
Maybe moving this to it's own method parameterized with its own method e.g., `void DebugBreakOrThrowException(string message)`.